components/fieldlabel/metadata/form.yml

```diff
@@ -7,25 +7,25 @@ examples:
     markup: |
       <form class="spectrum-Form">
         <div class="spectrum-Form-item">
-          <label for="fieldLabelExample-lifestory" class="spectrum-FieldLabel spectrum-FieldLabel--sizeM spectrum-Form-itemLabel spectrum-FieldLabel--left">Company Title</label>
+          <label for="fieldLabelExample-company-1" class="spectrum-FieldLabel spectrum-FieldLabel--sizeM spectrum-Form-itemLabel spectrum-FieldLabel--left">Company Title</label>
           <div class="spectrum-Form-itemField">
             <div class="spectrum-Textfield spectrum-Textfield--multiline">
-              <textarea id="fieldLabelExample-lifestory" placeholder="Enter your life story" name="field" class="spectrum-Textfield-input"></textarea>
```

Comment:
Is there any risk in removing the placeholders?

Reply:
Not that I see. If we need to capture in VRTs, it should be within Textfield. In general they were removed previously as they're [not intended to be used](https://spectrum.adobe.com/page/text-field/#Don%E2%80%99t-use-placeholder-text) or demonstrated.